Client/Ecosia/Settings/EcosiaDebugSettings.swift

```diff
@@ -181,11 +181,6 @@ class UnleashVariantResetSetting: HiddenSetting {
             }
             await MainActor.run {
                 self.settings.tableView.reloadData()
-                let alert = AlertController(title: "Unleash reset ✅", 
-                                            message: "The local Unleash cache has been wiped out",
-                                            preferredStyle: .alert)
-                alert.addAction(.init(title: "Ok", style: .default))
-                navigationController?.topViewController?.show(alert, sender: nil)
```

Comment:
I had to remove this alert because it was not properly working with our navigation.

Reply:
Thanks for spotting! I believe we can simply re-add that and instead of `show(::)` we can use `present(::)`. It should work as expected 💪 .

```
                let alert = AlertController(title: "Unleash reset ✅",
                                            message: "The local Unleash cache has been wiped out",
                                            preferredStyle: .alert)
                alert.addAction(.init(title: "Ok", style: .default))
                navigationController?.topViewController?.present(alert, animated: true)
```

Reply:
Good point, should have taken the time to understand why it did not work 🙈 will update it!